app/assets/stylesheets/application.scss

```diff
@@ -0,0 +1,44 @@
+@import "bootstrap-sprockets";
+@import "bootstrap";
+@import url('https://fonts.googleapis.com/css?family=Audiowide');
+
+body {
+  font-family: 'Audiowide', cursive;
+  font-size: 18px;
+}
+
+.navbar {
+  font-size: 16px;
+}
+
+.navbar-nav .open .dropdown-menu {
+  cursor: pointer;
+}
+
+.navbar-nav .open .dropdown-menu {
+  li:hover {
+    color: #9edce5;
```

Comment:
If you use color more than ones, you should use variable
https://htmlcolorcodes.com/tutorials/scss-color-variables/